state/state.go

```diff
@@ -1574,6 +1574,20 @@ func (st *State) AllApplications() (applications []*Application, err error) {
 	return applications, nil
 }
 
+// AllBindings return all endpoint - space bindings deployed.
+func (st *State) AllBindings() (bindings map[string]string, err error) {
```

Comment:
This needs to be allBindingsByApp()
Similar to the existing allUnitsByApp()

Reply:
ok. then this Allbindings shouldnt be here. It should be in state/endpoint_bindings.go.